examples/case_studies/probabilistic_matrix_factorization.ipynb

```diff
@@ -16,22 +16,34 @@
      "name": "stdout",
```

Comment:
<p>same aesthetic comment as above on the try except.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>